DXRFixes/DeusEx/Classes/ScriptedPawn.uc

```diff
@@ -107,23 +107,42 @@ function TakeDamageBase(int Damage, Pawn instigatedBy, Vector hitlocation, Vecto
 {
     local name baseDamageType;
     local DeusExPlayer p;
+    local Human h;
+    local bool wasOnFire;
 
     if (damageType == 'FlareFlamed') {
         baseDamageType = 'Flamed';
     } else {
         baseDamageType = damageType;
     }
 
+    wasOnFire=bOnFire;
+
     _TakeDamageBase(Damage,instigatedBy,hitLocation,momentum,baseDamageType,bPlayAnim);
 
+    if (!wasOnFire && bOnFire && instigatedBy==GetPlayerPawn()){
+        h = Human(GetPlayerPawn());
+        class'DXREvents'.static.MarkBingo(h.dxr,"IgnitedPawn");
```

Comment:
so you can quickly get this by using fire extinguishers on guys lol

couldn't we also detect this in their death event for non vanilla?